### PR TITLE
Add new Brasil OG image

### LINKED ISSUE #4080: Create OG image for Brasil
**Is your feature request related to a problem? Please describe.**
The current opengraph image for Brasil is just the 'BBC News' logo.  We should add a 'BBC News Brasil' opengraph image to Simorgh.

**Describe the solution you'd like**
Add the image supplied by UX

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #4080

**Overall change:**
Adds a new OG image for Brasil

**Code changes:**

- Adds image for Brasil
- Updates Brasil config to use new image
- Renames Scotland image directory to match
- Updates Scotland config to use new image path

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/portuguese.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ export const service = {
     brandName: 'BBC News Brasil',
     product: 'BBC News',
     serviceLocalizedName: 'Brasil',
-    defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/brasil.png',
+    defaultImage:
+      'https://news.files.bbci.co.uk/include/articles/public/portuguese/images/og/poster-1024x576.png',
```
Comment (sareh) [inline src/app/lib/config/services/portuguese.js]: This image is not just used for `og` opengraph, but also for `twitter:image` so I'd sugest naming the directory something like `/public/portuguese/images/metadata` instead.

**Review by sareh — CHANGES_REQUESTED** [review]
The image itself looks good - just the location should be updated.

File: src/app/lib/config/services/portuguese.js (hunk 1 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/portuguese.js]: OK

**Review by rhenshaw56 — APPROVED** [review]

**Review by sareh — APPROVED** [review]
Thanks for the update. Looks good.

**Pull request merged**